charts/kms-core/templates/_helpers.tpl

```diff
@@ -97,7 +97,7 @@ args:
 {{- include "proxyToEnclave"
       (dict "name" .name
             "image" .image
-            "from" (printf "TCP-LISTEN:%d,fork,reuseaddr" (int .port))
+            "from" (printf "TCP-LISTEN:%d,fork,nodelay,reuseaddr" (int .port))
```

Comment:
This is good, however, I think we should also add `nodelay` to the outgoing TCP proxies (`proxyFromEnclave`) just to be sure.